src/AvaloniaMath/TexFormulaParser.cs

```diff
@@ -64,27 +77,23 @@ internal static string[][] DelimiterNames
 
         private static void Initialize()
         {
-            
-            commands = new HashSet<string>
+            /* TODO: What?
+            //
+            // If start application isn't WPF, pack isn't registered by defaultTexFontParser
```

Comment:
Could you tell me what this means?

Reply:
In WPF application startup code, the `pack://` URL is registered. I believe this is required for us to properly render the fonts in a non-WPF environment (in cases when people use WPF-Math in ASP.NET apps, WinForms or something like that: e.g. in our unit tests, I think).

I don't think it's necessary in Avalonia code. How did it ended up there, anyway? Was this piece of code changed lately?

Reply:
Thanks! No, there were no changes. I will then delete it